### ISSUE TITLE
Continuous printing 'vase' toolpath for shells

### ISSUE THREAD

**Issue by yvanblanchard** [karl-nilsson/sse#43]
Hello,

I would like to test the continuous (vase) printing toolpath (for shell shapes).
I see a method called slicer::make_spiral_face,  that could do this job, but it does seem to be used today.
Could you confirmt this could help, and if yes how I could plug it for testing ?

Thank you.

**Comment by karl-nilsson** [comment]
> I would like to test the continuous (vase) printing toolpath (for shell shapes). I see a method called slicer::make_spiral_face, that could do this job, but it does seem to be used today. Could you confirmt this could help, and if yes how I could plug it for testing ?

Yes, that function is specifically intended for vase slicing. When I first started writing the library, I was a bit eager and tried to implement as many features I could think of, but didn't spend much time testing them. That function is one such example. I have no idea whether it works, or works correctly, and other major changes are necessary for vase slicing to output gcode.

I've since changed my workflow to prototyping and testing in python. You can find that repo here:
https://github.com/karl-nilsson/slicer_py
I'll have some free time in the coming weeks, so I'll try to prototype spiral/vase mode slicing in python.

Also I want to reiterate: this project is currently a proof of concept. Feel free to use it to generate gcode, but please don't try to print that gcode. I can't guarantee it won't do bad things to your printer.
